server/handlers/auth.ts

```diff
@@ -25,7 +25,7 @@ const authenticate = (
     passport.authenticate(type, (err, user) => {
       if (err) return next(err);
 
-      if (!user && isStrict) {
+      if (!env.SEARCH_ENABLED && !user && isStrict) {
```

Comment:
This is a big auth issue, if SEARCH_ENABLED is true everyone can pass auth and access handlers.